web/src/main/java/se/inera/intyg/rehabstod/service/sjukfall/GetSickLeaveSummaryServiceImpl.java

```diff
@@ -44,8 +44,18 @@ public GetSickLeaveSummaryServiceImpl(PuService puService, StatisticsCalculator
     @Override
     public SickLeaveSummary get() {
         final var sickLeaves = getActiveSickLeavesService.get(
-            new SickLeavesFilterRequestDTO(new ArrayList<>(), new ArrayList<>(), new ArrayList<>(), null, null, null, null),
-                INCLUDE_PARAMETERS);
+            new SickLeavesFilterRequestDTO(
+                    new ArrayList<>(),
```

Comment:
You can use Collections.emptyList() instead of creating ArrayList-objects.

Reply:
This produces an error when the doctorId is added for the filtration

Reply:
OK, I see... I've noticed "special" addDoctorId to the DTO. Suggest you remove it... There is not really a need for it and you can change the implementation in GetActiveSickLeavesServiceImpl.getRequest(...).